testutil/singlevm/verify.sh

```diff
@@ -281,32 +283,6 @@ checkEventStatus $event_counter "Unmapped"
 
 "$ciao_gobin"/ciao-cli external-ip list
 
-#Test for External IP Failures
-
-#Map failure
```

Comment:
@markdryan @mcastelino I added a new commit that allows the user to set the CNCI size in the configuration.yaml file for now. It's optional, and if it's not set, controller defaults will be used. We can remove this later once we have a programmatic way of determining the CNCI size. Take a look and tell me what you think.